R/tm_g_bivariate.R

```diff
@@ -327,16 +327,17 @@ ui_g_bivariate <- function(id, ...) {
         )
       )
     ),
-    forms = teal::get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
     pre_output = args$pre_output,
     post_output = args$post_output
   )
 }
 
 
 srv_g_bivariate <- function(id,
```

Comment:
Vignette crashes app and R session (!) when you click on "Wide and long datasets" 
![image](https://user-images.githubusercontent.com/15201933/187632912-81bcb052-7dcb-474c-9a63-e1fb66b8e2ac.png)

Reply:
Just reconfirming if I can reproduce as @Polkas couldn't

Reply:
ok so you need to click on wide and multiple long datasets
and then click on wide and long datasets
then it crashes
if you just click on wide and long first it doesn't

Reply:
crashes for me on main as well

Reply:
@nikolas-burkoff can you check again after my last commit?

Reply:
still crashing for me

Reply:
Edit: No it's fixed - thanks!

Reply:
Though it definitely still crashes if you jump between tabs too quickly as @Polkas noticed on main

Reply:
Edit: Now it crashes on the same tab if I move sequentially through the tabs...

Reply:
Any updates about this? Do we have a issue out of it or is it fixed already?